dev/com.ibm.ws.security.fat.common.mp.jwt/src/com/ibm/ws/security/fat/common/mp/jwt/actions/MpJwtFatActions.java

```diff
@@ -42,6 +42,9 @@ public class MpJwtFatActions extends JwtTokenActions {
     protected final String defaultPassword = MPJwtFatConstants.TESTUSERPWD;
 
     public String getJwtFromTokenEndpoint(String testName, String builderId, String builderUrlBase, String user, String pw) throws MalformedURLException, Exception {
+        // Sleep for 1 second to ensure created JWTs are unique; the iat claim for JWTs only has a granularity of seconds
+        Thread.sleep(1000);
```

Comment:
With the way that clock stuff works with Java, you may want to do 1100 to avoid timing issues.